data/json/items/armor.json

```diff
@@ -9909,5 +9909,26 @@
     "warmth": 10,
     "material_thickness": 1,
     "flags": [ "VARSIZE" ]
+  },
+  {
+    "id": "case_violin",
+    "type": "GENERIC",
+    "name": "violin case",
+    "name_plural": "violin case",
+    "description": "Useful to carry your precious musical instrument around protected from any harm.  In this case the case contains not an instrument but a Tommygun, a drum magazine and a forward grip, disassemble to get its contents.",
```

Comment:
You copied the description mentioning the Tommygun and the disassembly hint.

Reply:
Removed it.